src/main/kotlin/org/dif/common/JSON.kt

```diff
@@ -0,0 +1,21 @@
+package org.dif.common
+
+import com.nimbusds.jose.util.JSONObjectUtils
+
+interface JSON {
+    fun toJSONObject(): Any
+}
+
+class JSONObject(private val data: Map<String, Any>): JSON {
```

Comment:
Do we use these classes anywhere? It looks like we use `Map<String, Any>` instead.

Reply:
I suggest to decide it later